packages/service-library/src/servicelib/aiohttp/tracing.py

```diff
@@ -1,76 +1,65 @@
-""" Adds aiohttp middleware for tracing using zipkin server instrumentation.
+""" Adds aiohttp middleware for tracing using opentelemetry instrumentation.
 
 """
 import logging
-from typing import Iterable
 
-import aiozipkin as az
 from aiohttp import web
-from aiohttp.web import AbstractRoute
-from aiozipkin.aiohttp_helpers import (
-    APP_AIOZIPKIN_KEY,
-    REQUEST_AIOZIPKIN_KEY,
-    middleware_maker,
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import (
+    OTLPSpanExporter as OTLPSpanExporterHTTP,
 )
-from yarl import URL
+from opentelemetry.instrumentation.aiohttp_client import AioHttpClientInstrumentor
+from opentelemetry.instrumentation.aiohttp_server import AioHttpServerInstrumentor
+from opentelemetry.instrumentation.aiopg import AiopgInstrumentor
+from opentelemetry.instrumentation.requests import RequestsInstrumentor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from pydantic import AnyUrl
 
 log = logging.getLogger(__name__)
 
 
 def setup_tracing(
-    app: web.Application,
+    app: web.Application,  # pylint: disable=unused-argument
     *,
     service_name: str,
-    host: str,
-    port: int,
-    jaeger_base_url: URL | str,
-    skip_routes: Iterable[AbstractRoute] | None = None,
-) -> bool:
+    otel_collector_endpoint: AnyUrl | str | None,
```

Comment:
This function should be analogous to the `fastapi/tracing.py` counterpart. Let's keep the same function name and signature, i.e. this should be something like
```python
def setup_opentelemetry_instrumentation( app: web.Application, tracing_settings: TracingSettings, service_name: str):
   ... 
```